hack/ensure-kubectl.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 GOPATH_BIN="$(go env GOPATH)/bin/"
-MINIMUM_KUBECTL_VERSION=v1.15.0
+MINIMUM_KUBECTL_VERSION=v1.13.0
```

Comment:
How come we're dropping the min version?

Reply:
This belongs to the `revert later` commit but you are right, we only need to revert go version, not KUBECTL min version. Changed this.

As a side note to I will revert golang min version once I update prow images from 1.18 to 1.19 (because 1.19 is using go 1.15): https://github.com/kubernetes/test-infra/blob/9d2209384daf45ceef4016a4f1f04d06d37f48bf/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-periodics.yaml#L20